### PR TITLE
docs(accordion): add customization playgrounds

### PR DESCRIPTION
**Pull request by liamdebeasi**
https://ionic-docs-git-1310-customization-ionic1.vercel.app/docs/api/accordion#customization

A few notes:

1. The stackblitz "Advanced Expansion Styles" apps will not have the active accordion header turn blue. This is due to FW-1666.
2. The theming example for Angular won't look quite right. This is due to FW-1677.

### CODE REVIEW

File: docs/api/accordion.md [hunk 1]
```diff
@@ -103,11 +131,17 @@ If you would like to manage the icon yourself or use an icon that is not an `ion
...
+<Icons />
 
 ### Theming
 
-TODO Playground
+Since `ion-accordion` acts as a shell around the header and content elements, you can easily theme the accordion however you would like. You can theme the header by targeting the slotted `ion-item`. Since you are using `ion-item`, you also have access to all of the [ion-item CSS Variables](./item#css-custom-properties) and [ion-item Shadow Parts](./item#css-shadow-parts). Theming the content is also easily achieved by targeting the element that is in the `content` slot.
+
+import Theming from '@site/static/usage/accordion/customization/theming/index.md';
+
+<Theming />
```
Comment (amandaejohnston) [inline docs/api/accordion.md:144]: Did you mean to leave `expand="inset"` on these, or was that a copy-paste goof?

Comment (amandaejohnston) [inline docs/api/accordion.md:144]: Re: the Angular issue, you can always paste all the imports that were there originally? The popover styling playground does that, for example.

Comment (liamdebeasi) [reply docs/api/accordion.md:144]: Yes, I meant to keep that there. I wanted to show how to build on top of non-default expansion styles with theming.

Comment (liamdebeasi) [reply docs/api/accordion.md:144]: thanks, I'll do that for now

**Review by amandaejohnston — APPROVED** [review]

**Review by sean-perkins — APPROVED** [review]

**Pull request merged**